vmm/src/vmm_config/boot_source.rs

```diff
@@ -2,8 +2,30 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use std::fmt::{Display, Formatter, Result};
+use std::fs::File;
```

Comment:
nit: reorder alphabetically

Reply:
Unfortunately I fail to see what is not in alphabetical order :( Plus, `cargo fmt` puts `use`s in the correct order anyhow.